src/local-development/helpers/get-origin-object.ts

```diff
@@ -0,0 +1,36 @@
+import { LocalAppOrigin, MakecomappJson } from '../types/makecomapp.types';
+
+/**
+ * Returns the origin object freom makecomapp.json.
+ * Used for editation this object for case, when input `origin` can be a clone of this object.
```

Comment:
Please reformulate this sentence a bit...

Reply:
Improving ...